tests/integrations/tso/client_test.go

```diff
@@ -197,6 +197,37 @@ func (suite *tsoClientTestSuite) TestGetTSAsync() {
 	wg.Wait()
 }
 
+func (suite *tsoClientTestSuite) TestGetMinTS() {
```

Comment:
Is it possible to write a test to ensure that the min ts obtained must be the smallest of all keyspace groups rather than simply comparing with the previous min ts?

Reply:
it's covered in this test case.

Reply:
```
					// Now we check whether the returned ts is the minimum one
					// among all keyspace groups, i.e., the returned ts is
					// less than the new timestamps of all keyspace groups.
					for _, client := range suite.clients {
						physical, logical, err := client.GetTS(suite.ctx)
						suite.NoError(err)
						ts := tsoutil.ComposeTS(physical, logical)
						suite.Less(minTS, ts)
					}
```

Reply:
Got it, my bad.